apps-rendering/src/renderer.test.ts

```diff
@@ -223,20 +225,23 @@ const audioAtom = (): BodyElement => ({
 const render = (element: BodyElement): ReactNode[] =>
 	renderAll(mockFormat, [element]);
 
+const renderElements = (elements: BodyElement[]): ReactNode[] =>
+	renderAll(mockFormat, elements);
+
```

Comment:
Maybe not one for this PR, but it might be worth considering a test helpers file at some point? I think this and the `textElements` function above are similar to the ones in the ads tests?

Reply:
Great suggestions. Since it didn't seem to be a lot of work, I did it as part of this PR 👍